web_tree_background_color/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,5 @@
+This module allows you to dynamically set the **background** color
+of list view rows, according to python expressions; similarly to the
+`standard odoo feature to set the text color`_.
+
+.. _standard odoo feature to set the text color: https://www.odoo.com/documentation/10.0/reference/views.html#lists
```

Comment:
worth to mention that the module simply adds extra classes, so if you want to use custom colors you can define your own class + LESS file to style it.

Reply:
I didn't even think of that, that's great to do some advanced stuff. Will surely add to the README.